Makefile

```diff
@@ -39,6 +39,16 @@ download:
 	@echo Download go.mod dependencies
 	@go mod download
 
+## install-tools: Installing tools from tools.go
+install-tools: download
```

Comment:
The `assets` Makefile target should have this as a prerequisite or some check verifying that go-bindata exists.

Due to I originally didn't execute install-tools I received this error:

```
msoriano@localhost:~/go/src/github.com/3scale/3scale-operator (monitoring-tools)$ make assets
Generate Go embedded assets files by processing source
go generate github.com/3scale/3scale-operator/pkg/assets
pkg/assets/assets.go:9: running "go-bindata": exec: "go-bindata": executable file not found in $PATH
make: *** [Makefile:50: assets] Error 1
```

Reply:
:+1: